braindecode/datautil/serialization.py

```diff
@@ -114,12 +116,33 @@ def _load_signals_and_description(path, preload, is_raw, ids_to_load=None):
 
 
 def _load_signals(fif_file, preload, is_raw):
+
+    # Reading the raw file from pickle if it has been save before.
+    # The pickle file only contain the raw object without the data.
+    pkl_file = fif_file.with_suffix(".pkl")
+    if pkl_file.exists():
```

Comment:
maybe we can add some logging that we'are loading from pickle? I imagine situation in which you modify raw files and you're stuck with old pickle files.